main.go

```
@@ -0,0 +1,142 @@
+		args = append(args, "-p", fmt.Sprintf("%d:%d", port, port))
+	}
+	args = append(args, pod.Image)
+	cmd := exec.Command("docker", args...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	return cmd.Run()
+}
+
+func stopContainer(name string) {
+	cmd := exec.Command("docker", "rm", "-f", name)
+	cmd.Run()
```

golangci-lint (1.64.8)

Error return value of `cmd.Run` is not checked

(errcheck)